specification/trace/semantic_conventions/http.md

```diff
@@ -194,6 +194,21 @@ Following attributes MUST be provided **at span creation time** (when provided a
 
 Note that in some cases host and port identifiers in the `Host` header might be different from the `server.address` and `server.port`, in this case instrumentation MAY populate `Host` header on `http.request.header.host` attribute even if it's not enabled by user.
 
+### HTTP client span duration
+
+HTTP client spans SHOULD start sometime before the first request byte is sent. This may or may not include connection time.
+
+HTTP client spans SHOULD end be sometime after the HTTP response headers are read (or when they fail to be read).
+
+If there is any possibility for application code to not fully read the HTTP response
+(and for the HTTP client library to then have to clean up the HTTP response asynchronously),
+the HTTP client span SHOULD be ended earlier
```

Comment:
"earlier" than the response headers being read? This does not make sense anymore I think.